OpenSim/Simulation/Model/CoordinateLimitForce.h

```diff
@@ -235,9 +237,10 @@ OpenSim_DECLARE_CONCRETE_OBJECT(CoordinateLimitForce, Force);
 
     // Smooth step functions for continuous transition from no stiffness and 
     // damping to constant values beyond the limits. These are heap allocated
-    // and owned here so must be deleted in destructor.
-    SimTK::ReferencePtr<SimTK::Function::Step> _upStep;
-    SimTK::ReferencePtr<SimTK::Function::Step> _loStep;
+    // and owned here, but we don't want these functions to be copied if a
+    // CoordinateLimitForce is copied.
```

Comment:
> I feel bad now that you modified the Function classes

No worries! I like that the Function classes got improved regardless of whether they are used now.